tests/scripts/booking/test_update_confirmation_date_from_existing_bookings.py

```diff
@@ -0,0 +1,82 @@
+import datetime
+from collections import namedtuple
+
+import pytest
+
+from pcapi.core.bookings import factories as bookings_factories
+from pcapi.core.offers import factories as offers_factories
+from pcapi.core.users.factories import UserFactory
+from pcapi.scripts.booking.update_confirmation_date_from_existing_bookings import _select_all_event_bookings_without_confirmation_date, _update_target_bookings, \
+    fill_missing_confirmation_dates_of_all_event_bookings_without_confirmation_date, \
+    fill_missing_confirmation_dates_of_event_bookings_without_confirmation_date_not_cancelled_not_used, get_all_event_bookings_without_confirmation_date_count, \
+    get_event_bookings_without_confirmation_date_not_cancelled_not_used_count
+
+
+@pytest.mark.usefixtures("db_session")
+class UpdateConfirmationDateFromExistingBookingsTest:
+    @pytest.fixture(name='test_data')
+    def test_data_fixture(self):
+        last_year = datetime.datetime(2019, 7, 14, 3)
+        today_at_2 = datetime.datetime(2020, 11, 5, 2)
+        tomorrow_at_13 = datetime.datetime(2020, 11, 6, 13)
+        user = UserFactory()
+        event_stock = offers_factories.StockFactory(beginningDatetime=tomorrow_at_13, price=0)
```

Comment:
```suggestion
        event_stock = StockFactory(beginningDatetime=tomorrow_at_13, price=0)
```
Pourquoi avoir un préfixe ?

Reply:
C'est une question de style d'import, j'aime bien repérer dans quel module se situe la classe, surtout quand ce n'est pas évident dans son nom; je peux complètement le changer.